internal/connectors/engine/activities/storage_payments_delete_from_psu_id_and_connector_id.go

```diff
@@ -0,0 +1,19 @@
+package activities
+
+import (
+	"context"
+
+	"github.com/formancehq/payments/internal/models"
+	"github.com/google/uuid"
+	"go.temporal.io/sdk/workflow"
+)
+
+func (a Activities) StoragePaymentsDeleteFromPSUIDAndConnectorID(ctx context.Context, psuID uuid.UUID, connectorID models.ConnectorID) error {
```

Comment:
if we add the relation + on delete cascade on the payment table, can we avoid adding the temporal activities?

Reply:
just to be clear, we want ot delete all payments in dedicted API calls, not when:

* we delete PSUID => deleting all related payments via ON DELETE should od the trick
* we delete the connector => again, deleting all payments via the connectorID should do the trick (we're fine to ignore PSUID I think here)
* we delete the connection => as above, if the connection is done we 💥 the payments and we ignore PSUID & connectorID

I guess the question is "do we ever delete all payments, beside when we delete connector, connection or PSU"?